catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/java/org/codice/ddf/registry/admin/plugin/AdminRegistryPollerServiceBean.java

```diff
@@ -0,0 +1,300 @@
+        } catch (MalformedObjectNameException e) {
+            LOGGER.error("Unable to create Admin Registry Poller Service MBean with name [{}].",
+                    AdminRegistryPollerServiceBean.class.getName() + SERVICE_NAME,
+                    e);
+        }
+        objectName = objName;
+    }
+
+    public void init() {
+        try {
+            try {
+                mBeanServer.registerMBean(this, objectName);
+                LOGGER.info("Registered Admin Registry Poller Service MBean under object name: {}",
+                        objectName.toString());
+            } catch (InstanceAlreadyExistsException e) {
+                // Try to remove and re-register
+                mBeanServer.unregisterMBean(objectName);
+                mBeanServer.registerMBean(this, objectName);
+                LOGGER.info("Re-registered Admin Registry Poller Service MBean");
+            }
+        } catch (Exception e) {
+            LOGGER.error("Could not register MBean [{}].", objectName.toString(), e);
+        }
+    }
+
+    public void destroy() {
+        try {
+            if (objectName != null && mBeanServer != null) {
+                mBeanServer.unregisterMBean(objectName);
+                LOGGER.info("Unregistered Admin Registry Poller Service Service MBean");
+            }
+        } catch (Exception e) {
+            LOGGER.error("Exception unregistering MBean [{}].", objectName.toString(), e);
+        }
+    }
+
+    @Override
+    public boolean registryStatus(String servicePID) {
+        try {
+            List<Source> sources = helper.getRegistrySources();
+            for (Source source : sources) {
+                if (source instanceof ConfiguredService) {
+                    ConfiguredService cs = (ConfiguredService) source;
+                    try {
+                        Configuration config = helper.getConfiguration(cs);
+                        if (config != null && config.getProperties()
+                                .get("service.pid")
+                                .equals(servicePID)) {
+                            try {
+                                return source.isAvailable();
+                            } catch (Exception e) {
+                                LOGGER.warn("Couldn't get availability on registry {}: {}",
+                                        servicePID,
+                                        e);
+                            }
+                        }
+                    } catch (IOException e) {
+                        LOGGER.warn("Couldn't find configuration for source '{}'", source.getId());
+                    }
+                } else {
+                    LOGGER.warn("Source '{}' not a configured service", source.getId());
+                }
+            }
+        } catch (InvalidSyntaxException e) {
+            LOGGER.error("Could not get service reference list");
+        }
+
+        return false;
```

Comment:
I'm not seeing the value of the return type being a boolean here since you only ever return false.